content/operate/rs/clusters/logging/rsyslog-logging/_index.md

```diff
@@ -29,7 +29,7 @@ All log entries displayed in the Cluster Manager UI are also written to `syslog`
 
 Log entries are categorized into events and alerts. Both types of entries appear in the logs, but alert log entries also include a boolean `"state"` parameter that indicates whether the alert is enabled or disabled.
 
-Log entries include information about the specific event that occurred. See the log entry tables for [clusters]({{< relref "/operate/rs/clusters/logging/rsyslog-logging/cluster-events" >}}), [databases]({{< relref "/operate/rs/clusters/logging/rsyslog-logging/bdb-events" >}}), [nodes]({{< relref "/operate/rs/clusters/logging/rsyslog-logging/node-events" >}}), and [users]({{< relref "/operate/rs/clusters/logging/rsyslog-logging/user-events" >}}) for more details.
```

Comment:
Line 14 of this file duplicates the description: so I think we should change it or remove it?